agent/stats/queue_test.go

```diff
@@ -314,6 +314,18 @@ func validateNetStatsSet(t *testing.T, netStats *ecstcs.NetworkStatsSet, queueLe
 	assert.Equal(t, int64(queueLen), *netStats.TxErrors.SampleCount, "incorrect TxErrors sampleCount")
 	assert.Equal(t, int64(0), *netStats.TxErrors.Sum, "incorrect TxErrors sum")
 	assert.Equal(t, int64(0), *netStats.TxErrors.OverflowSum, "incorrect TxErrors overlfowSum")
+
+	assert.NotNil(t, *netStats.RxBytesPerSecond, "incorrect RxBytesPerSecond set")
+	assert.NotEqual(t, math.MaxFloat64, *netStats.RxBytesPerSecond.Min, "incorrect RxBytesPerSecond min")
```

Comment:
why are these tests not asserting the actual value that we expect for RxBytesPerSecond and TxBytesPerSecond?

Reply:
The actual float values are quite messy to assert. That is why just made sure that they are not empty and not not nil.

Reply:
you can use assert.InDelta to assert float values are within some delta of each other: https://pkg.go.dev/github.com/stretchr/testify/assert?tab=doc#InDelta

Reply:
Fixed it